### DISCUSSION TITLE
Did you mean functionality in PnP Search

### DISCUSSION THREAD

**Discussion by AtlasPark77** [microsoft-search/pnp-modern-search#2251] [Q&A] [answered]
@wobba 
Hi Team, first post for me, so I apologies if I have asked this question in the wrong place.

We're currently using PnP version 4.6.1 and have activated the "Did you mean" functionality that is working a treat.  The question I have is can the "Did you mean" dictionary be added to?  We have a requirement to add in specific spelling mistakes for acronyms and other company related terms  

![image](https://user-images.githubusercontent.com/109932963/180681271-91885610-baa1-4b06-9f6e-1d3fc7f407bd.png)

Kind Regards,

Atlas

**Comment by FranckyC** [comment, marked as answer]
Hi @AtlasPark77, I don't think you can control that unfortunately :/

**Reply by AtlasPark77** [reply]
Thank you for your response.  We may need to look into another way of doing this
